infrastructure/batch/job_queue.tf

```diff
@@ -3,11 +3,9 @@ resource "aws_batch_job_queue" "scpca_portal_project" {
   state    = "ENABLED"
   priority = 1
 
-  # compute_environments = [] array is deprecated
-  compute_environment_order {
-    order               = 1
-    compute_environment = aws_batch_compute_environments.scpca_portal_project.arn
-  }
+  compute_environments = [
+    aws_batch_compute_environment.scpca_portal_project.arn,
+  ]
```

Comment:
`compute_environments` is currently deprecated. The recommended approach is using `compute_environment_order`. Using `compute_environment_order` necessitates upgrading to aws provider version "5.40", which I ran into bugs while trying to implement. In the interest of time I kept `compute_environments` for now, but this should be updated in the future.